tests_integration/test_installation.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+import subprocess
+from pathlib import Path
+
+from snowcli import __about__
+
+
+# @pytest.mark.integration
+def test_installation(venv_path, test_root_path, snapshot):
```

Comment:
Should we add a test for `snow streamlit init` that uses templates resources? This will help us make sure resources are distributed

Reply:
Added